src/reducers/index.ts

```diff
@@ -247,9 +248,24 @@ const rssPostsReducer = (rssPosts = List<Post>(), action: Actions): List<Post> =
     return rssPosts;
 };
 
+const removeItem = <T>(list: List<T>, predicate: (value?: T, index?: number, iter?: Iterable.Indexed<T>) => boolean): List<T> => {
+    const ind = list.findIndex(predicate);
+    if (ind >= 0) {
+        return list.remove(ind);
+    } else {
+        return list;
+    }
+};
+
 const localPostsReducer = (localPosts = defaultLocalPosts, action: Actions): List<Post> => {
     switch (action.type) {
         case 'ADD-POST': {
+            if (localPosts.findIndex(post => post != null && post._id === defaultPost1._id) >= 0) {
```

Comment:
I see a potential performance problem here. Anytime you add a post, this will iterate through all the posts minimum once. This also means that it will get much worse after you have many posts.
In theory it would be enough to trigger the condition when the action.payload.post._id  equals defaultLocalPosts.size, no?